components/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/BPELDeployer.java

```diff
@@ -106,7 +106,6 @@ public void initialize(List<Parameter> parameterList) throws WorkflowImplExcepti
                     (bpsProfileName, tenantId);
         }
         htName = processName + BPELDeployer.Constants.HT_SUFFIX;
-
```

Comment:
let's remove this unintended changes

Reply:
addressed the comment ref:https://github.com/wso2-extensions/identity-workflow-impl-bps/pull/70/commits/641aaf120ea3a609b753a77ae028dae66a9ffa98